SIGS.md

```diff
@@ -0,0 +1,6 @@
+<!-- sigs -->
+| Name | Owners | Project | Board | Areas | Status | Directories | Notes |
```

Comment:
I wonder if we need directories here - I'd be happy to help automating codeowners generation and this is where this info would be necessary.

Reply:
ah right I see. I guess doesn't hurt to get the quick overview on where to find stuff?